zalo_sdk/oa/Client.py

```diff
@@ -69,12 +72,12 @@ def get_access_token_from_authorization_code(self, authorization_code):
             'code': authorization_code,
             'app_id': self.app_id,
             'grant_type': 'authorization_code',
-            'code_verifier': 'your_code_verifier'
+            'code_verifier': code_verifier
```

Comment:
We need to send code_verifier since we send the state in step2
<img width="871" alt="image" src="https://user-images.githubusercontent.com/129479557/233318097-586f11af-c4f0-48e5-a72a-63ccece038e3.png">